controls/views.py

```diff
@@ -400,10 +408,12 @@ def get_context_data(self, **kwargs):
         if self.request.user.has_perm('view_system', system):
             # Retrieve primary system Project
             # Temporarily assume only one project and get first project
-            project = system.projects.first()
+            project = Project.objects.filter(system_id=system.id).first()
```

Comment:
Just curious: is this a performance improvement?  It looks functionally equivalent to the line it is replacing.

Reply:
The .first() on line 441 is superfluous and can (should) be removed.

The question of whether querying for a Project by filter on the system id is more performant than assuming that there is only one Project associated with system and getting the first one, I don't know.

Reply:
This change is superfluous and was made when trying to solve a problem that ultimately was solved at the template level. Removing...